proposals/lip-0041.md

```diff
@@ -250,16 +452,15 @@ The Auth module does not execute any logic during the genesis state finalization
 
 #### Transaction Verification
```

Comment:
I think this section should be updated as the signature verification is not the same any more as in LIP 17 as there is no exceptional case any more for the multisignature registration transaction. I think it would be best to have one new clean function here. The `verifySignatures` endpoint could then also point to the function.

Reply:
I rewrote the specs from scratch